### PR TITLE
feat: Handle and show errors when uploading file

### LINKED ISSUE #2167: [Creation] Handle and show errors when uploading file
## 🎉 Description

Show the following errors:
- File too big
- The file doesn't have animation (so it's not an emote)

* [ ] 🐛 This is a bug report.
* [X] 📈 This is a feature request.

### LINKED ISSUE #2228: [BUG] The 'CreateSingleItemModal' dimmed too much when setting the price to emotes
## 🎉 Description

When creating an emote using the `CreateSingleItemModal` at the `set price step`, the background turns too black because we're opening the modal: `EditPriceAndBeneficiaryModal`.

It should look like this:

* [x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

The idea is to remove the dimmed background on the `EditPriceAndBeneficiaryModal` modal when it opens from the `CreateSingleItemModal` modal.

### PR DESCRIPTION
**Pull request by cyaiox**
This PR updates handle and show errors when uploading a file.

Error: Max model file size
Screenshot 2022-11-17 at 17 41 26

Error: Max emote duration
Screenshot 2022-11-17 at 17 42 10

Closes #2167 
Closes #2228

### CODE REVIEW

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx [hunk 1]
```diff
@@ -245,6 +278,7 @@ export default class ImportStep extends React.PureComponent<Props, State> {
             onDropRejected={this.handleDropRejected}
           />
           {wearablePreviewComponent}
+          <WearablePreview id="wearable-preview" disableBackground disableAutoRotate />
```
Comment (juanmahidalgo) [inline src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:281]: Are we going to render 2 wearable preview components now? The one above will have the `id` `thumbnail-picker`, which is different from this new one but do we need both?

File: src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.tsx [hunk 2]
```diff
@@ -129,12 +129,12 @@ export default class EditPriceAndBeneficiaryModal extends React.PureComponent<Pr
...
 
   render() {
-    const { name, isLoading, onClose, onSkip } = this.props
+    const { name, isLoading, mountNode, onClose, onSkip } = this.props
     const { isFree, isOwnerBeneficiary, price = '' } = this.state
     const beneficiary = this.getBeneficiary()
 
     return (
-      <Modal name={name} size="tiny" onClose={onClose}>
+      <Modal name={name} size="tiny" onClose={onClose} mountNode={mountNode}>
```
Comment (juanmahidalgo) [inline src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.tsx:137]: Just out of curiosity, why do we need the `mountNode` now? Is it related to the main issue or is it solving another thing?

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx [hunk 3]
```diff
@@ -100,7 +108,21 @@ export default class ImportStep extends React.PureComponent<Props, State> {
       [modelPath]: file
     }
 
-    return this.processModel(modelPath, contents)
+    const { model, contents: proccessedContent, type } = await this.processModel(modelPath, contents)
+
+    if (type === ItemType.EMOTE) {
+      const data = await getItemData({
+        wearablePreviewController: WearablePreview.createController('wearable-preview'),
```
Comment (juanmahidalgo) [inline src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:115]: is there a way we can re-use the controller that we're receiving by parameter? It's wrapped in a React component but we could change how it's being passed down to be just the controller and wrap it here. WDYT?

File: src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.tsx (hunk 2 above)
Comment (cyaiox) [reply src/components/Modals/EditPriceAndBeneficiaryModal/EditPriceAndBeneficiaryModal.tsx:137]: it's used to solve #2228. Having this `mountNode`, it's easier to know when more than one modal is open and apply styles.

File: src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx (hunk 3 above)
Comment (cyaiox) [reply src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx:115]: sounds good! I'll refactor the wrapped component to use here :D

File: .husky/pre-commit [hunk 4]
```diff
@@ -10,7 +10,7 @@ fi
 echo "Running prettier"
 npm run pre-commit:fix:prettier -- $FILES
 
-TS_FILES=$(echo $FILES | { grep -E '(js|ts|tsx)$' || true; })
+TS_FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g' | { grep -E '(js|ts|tsx)$' || true; })
```
Comment (cyaiox) [inline .husky/pre-commit:13]: Change not related to this PR. It fixes an `eslint` error when applying to incorrect files.

**Review by juanmahidalgo — APPROVED** [review]
this looks great, good job @cyaiox ! 🚀

**Pull request merged**